src/View/StringTemplate.php

```diff
@@ -178,7 +179,7 @@ protected function _compileTemplates(array $templates = []): void
         foreach ($templates as $name) {
             $template = $this->get($name);
             if ($template === null) {
-                $this->_compiled[$name] = [null, null];
+                throw new InvalidArgumentException("String template '$name' is not valid.");
```

Comment:
If you use normal fencing `, then you can use normal PHP quotes ' around it.

Reply:
I am never really sure when to use `sprintf()` or what I did above. Sometimes we do 

* `'String ' . $url . ' did not parse'` or 
* `"Unknown type alias '$type'."` or 
* `sprintf('String template `%s` is not valid.', $name)` 

Do we want to get something consistent here or is it always the good old "it depends"?